bgpd/bgp_evpn_vty.c

```diff
@@ -3192,6 +3192,21 @@ static void evpn_show_all_routes(struct vty *vty, struct bgp *bgp, int type,
 	}
 }
 
+int bgp_evpn_show_all_routes(struct vty *vty, struct bgp *bgp, int type,
```

Comment:
Does `json` need to be free'd after getting a new object here?

Reply:
It's freed by vty_json().